teachertool/src/components/CriteriaTable.tsx

```diff
@@ -0,0 +1,69 @@
+/// <reference path="../../../built/pxtblocks.d.ts"/>
+
+import { useContext } from "react";
+import { Strings } from "../constants";
+import { AppStateContext } from "../state/appStateContext";
+import { getCatalogCriteriaWithId } from "../state/helpers";
+import { removeCriteriaFromRubric } from "../transforms/removeCriteriaFromRubric";
+import { CriteriaInstance } from "../types/criteria";
+import { classList } from "react-common/components/util";
+import { Button } from "react-common/components/controls/Button";
+import css from "./styling/CriteriaTable.module.scss";
+import React from "react";
+
+interface CriteriaInstanceDisplayProps {
+    criteriaInstance: CriteriaInstance;
+}
+const CriteriaInstanceRow: React.FC<CriteriaInstanceDisplayProps> = ({ criteriaInstance }) => {
+    const catalogCriteria = getCatalogCriteriaWithId(criteriaInstance.catalogCriteriaId);
+    if (!catalogCriteria) {
+        return null;
+    }
+
+    return catalogCriteria ? (
+        <div className={css["criteria-instance-display"]} role="row" tabIndex={0}>
```

Comment:
Why is this tabbable?

Reply:
Originally thought it was needed for table navigation, but I looked into it a little more and it seems like the expectation is to use ctrl+alt_arrows to navigate a table which works without this, so I'll remove.